proto/shentu/bounty/v1/tx.proto

```diff
@@ -29,7 +27,7 @@ message MsgCreateProgram {
   [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"submission_end_time\""];
 
   string                   creator_address  = 4 [(gogoproto.moretags) = "yaml:\"creator_address\""];
-  google.protobuf.Any      encryption_key   = 5 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey"];
```

Comment:
wrong key

Reply:
changes encryption_key type bytes to any